shpc/main/modules/__init__.py

```diff
@@ -229,7 +229,8 @@ def check_symlink(self, module_dir):
         if os.path.exists(symlink_path) and not utils.confirm_action(
             "%s already exists, are you sure you want to overwrite?" % symlink_path
         ):
-            sys.exit(0)
+            return False
```

Comment:
Hmm I don't think I agree with this change - if the user aborts we need to stop.

Reply:
With `sys.exit(0)` in place, the module won't be installed at all. One wouldn't be able to install say both `ruby:3.1.0` and `ghcr.io/autamus/ruby:3.1.0` at the same time. I thought that `shpc` was meant to still allow the same software+version to exist in multiple namespaces, and the user would essentially choose which one to put as symlink.

Reply:
We don't currently allow that - you can only install to one namespace (and then would be able to symlink to a name at the lowest level). 

I do think we want to exit here, given the user has explicitly said "no I don't want to overwrite," and they can decide to install the module not as a symlink. I think it's easier to press up and reproduce the install command without the symlink flag as opposed to seeing it install, needing to control C or otherwise try to undo it.

Reply:
Assuming people used the `--symlink-tree` command-line option. If `symlink_tree` is enabled globally in `settings.yaml`, one would have to edit the latter, install the software, and edit the config file back

Reply:
That’s a good point. But also if they are requiring symlink tree I’d argue a namespace conflict in the tree would prevent them from installing, period. I would rather see the opposite flag for this case, —no-symlink-tree, than for an install to continue that a person has explicitly said no to. Or I’d rather see a confirm choose with a numbered set of options:

1. continue install without symlink

2. abort install

Reply:
I added `--no-symlink-tree`